server/src-lib/Hasura/GraphQL/Transport/HTTP.hs

```diff
@@ -43,21 +44,20 @@ import qualified Network.Wai.Extended                   as Wai
 class Monad m => MonadExecuteQuery m where
   executeQuery
     :: GQLReqParsed
-    -> [R.QueryRootFldUnresolved]
```

Comment:
This needs to be added back in some form, please.

Reply:
@paf31 I believe @codingkarthik added back something. Can you verify if the new version is adequate for the intended purpose? The type `QueryRootFldUnresolved` has been replaced by `QueryRootField UnpreparedValue`.

Reply:
Yes, I have a branch which works with the new version, thanks for checking.